packages/conf-bap-llvm/conf-bap-llvm.1.1/opam

```diff
@@ -8,12 +8,23 @@ license: "BSD"
 build: [
   ["bash" "-ex" "configure"]
 ]
-depends: ["conf-which" {build}]
+depends: [
+  "ocaml"
+  "conf-which" {build}
+]
 depexts: [
-  ["llvm-3.4-dev"] {os-distribution = "debian"}
-  ["llvm-3.8-dev"] {os-distribution = "ubuntu"}
-  ["llvm-3.8"] {os = "macos" & os-distribution = "macports"}
-  ["homebrew/versions/llvm38"] {os = "macos" & os-distribution = "homebrew"}
+
+  # debian
+  ["llvm-4.0-dev"] {os-distribution = "debian"}
+
+  # ubuntu
+  ["llvm-3.8-dev"] {os-distribution = "ubuntu" & os-version = "14.04"} #trusty
+  ["llvm-4.0-dev"] {os-distribution = "ubuntu" & os-version = "16.04"} #xenial
+  ["llvm-4.0-dev"] {os-distribution = "ubuntu" & os-version = "18.04"} #bionic
+
+  # macos
+  ["llvm-4.0"] {os = "macos" & os-distribution = "macports"}
+  ["llvm@4"]   {os = "macos" & os-distribution = "homebrew"}
+
 ]
-synopsis: "Virtual package relying on llvm library installation for BAP"
-extra-files: ["configure" "md5=8b655d55580e497b799fb97d047dc15d"]
+extra-files: ["configure" "md5=72433d175e042cf95d85eb5fe7fabe4c"]
```

Comment:
All conf packages must have the `conf` flag set, see the [manual][1]

[1]: https://opam.ocaml.org/doc/Manual.html#opamfield-flags